webpack.config.js

```diff
@@ -0,0 +1,81 @@
+const path = require('path');
+
+module.exports = {
+    mode: 'development',
+    entry: './src/index.js',
+    devtool: "source-map",
+    output: {
+        filename: 'main.js',
+        path: path.resolve(__dirname, 'dist'),
+    },
+    externals: {
+        angular: 'angular',
+        lodash: 'lodash'
+    },
+ 
+    module: {
+        rules: [
+            {
+                test: /all\.less$/,
+                use: [
+                    {
+                        loader: 'style-loader', // creates style nodes from JS strings
+                    },
+                    {
+                        loader: 'css-loader', // translates CSS into CommonJS
+                    },
+                    {
+                        loader: 'less-loader', // compiles Less to CSS
+                        options: {
+                            lessOptions: {
+                                javascriptEnabled: true
+                            }
+                        }
+                    }
+                ]
+            },
+            {
+                test: /\.pug$/i,
+                exclude: /assets\/index\.pug$/,
+                use: [
+                    {
+                        loader: 'apply-loader',
+                    },
+                    {
+                        loader: 'pug-loader'
+                    
+                    },
+                ],
+            },
+            {
+                test: /\.(png|jpe?g|gif)$/i,
+                use: [
+                    {
+                        loader: 'url-loader',
+                    },
+                ],
+            },
+            {
+                test: /\.svg$/,
+                loader: 'svg-inline-loader'
+            },
+            {
+                test: require.resolve('jquery'),
+                loader: 'expose-loader',
+                options: {
+                    exposes: '$',
+                },
+            },
+            {
+                test: /\.(woff(2)?|ttf|eot|svg)(\?v=\d+\.\d+\.\d+)?$/,
+                use: [
+                    {
+                        loader: 'file-loader',
+                    },
+                ],
+            }
+        ],
+    },
+    
+
+};
```

Comment:
Since this is not an SPA, can we remove Webpack? We can remove a little bit of confusion and potentially save some time when doing `npm install`.

Reply:
yes we can do it ;)